src/plugins/api/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "api",
```

Comment:
We have a convention of using `/api/${plugin.name}` for a plugins routes. I feel like we should pick a better name for this plugin and then follow the naming convention everyone else does.